Fix wording >  introduction_to_grid.md (#27494)
TheOdinProject/curriculum: intermediate_html_css/grid/introduction_to_grid.md

[![flex-exercise-desired-outcome.png](https://i.postimg.cc/vZ81HMkB/flex-exercise-desired-outcome.png)](https://github.com/TheOdinProject/css-exercises/tree/main/flex/07-flex-layout-2)

> It challenged your understanding of Flexbox, but that's how you learn and grow as a developer."

Reviewer: Nit: This reads a little better to me: 

```suggestion
That was a challenging one! but that's how you learn and grow as a developer."
```

Author: solved
